packages/cli/src/generate/generators/app/create-app.ts

```diff
@@ -21,6 +21,46 @@ function isYarnInstalled() {
   }
 }
 
+function validateProjectName(name: string) {
+  let flag = 0;
+  const specialChars = ['!',
```

Comment:
Could you add a comment here with the link of the npm page where the characters were found?